frontend/src/components/layout/Sidepanel.vue

```diff
@@ -57,80 +57,107 @@
+                        break;
+                    case 'page-views':
+                        menu.Audience.push({
                             text: 'Page Views',
                             route: 'page-views',
-                        },
-                        {
+                        });
+                        break;
+                    case 'geo-location':
+                        menu.Audience.push({
                             text: 'Geo Location',
                             route: 'geo-location',
-                        },
-                    ]
-                },
-                {
-                    icon: '/assets/icons/settings.svg',
-                    text: 'Behaviour',
-                    route: 'behaviour',
-                    sublinks: [
-                        {
+                        });
+                        break;
+                    case 'behaviour':
+                        menu.Behaviour.push({
                             text: 'Visitors flow',
                             route: 'visitors-flow',
-                        }
-                    ]
-                },
-                {
-                    icon: '/assets/icons/speed.svg',
-                    text: 'Speed Overview',
-                    sublinks: [
-                        {
+                        });
+                        break;
+                    case 'page-timings':
+                        menu.Overview.push({
                             text: 'Page Timings',
                             route: 'page-timings',
-                        },
-                        {
+                        });
+                        break;
+                    case 'error-reports':
+                        menu.Overview.push({
                             text: 'Error Reports',
                             route: 'error-reports',
-                        },
-                    ]
-                },
-                {
-                    icon: '/assets/icons/settings.svg',
-                    text: 'Settings',
-                    route: 'settings',
-                    sublinks: [
-                        {
-                            text: 'User',
-                            route: 'user-update',
-                        },
-                        {
-                            text: 'Team',
-                            route: 'team',
-                        },
-                        {
-                            text: 'Website',
-                            route: 'websiteinfo',
-                        },
-                    ]
-                },
-            ]
-        })
+                        });
+                        break;
+                    }
+                });
+
+                return [
+                    {
+                        icon: '/assets/icons/home.svg',
+                        text: 'Home',
+                        route: 'dashboard'
+                    },
+                    {
```

Comment:
maybe it's better to hide section entirely if there is no sublinks available? just a thought, since such items won't do anything on click